lib/modules/platform/gitlab/index.ts

```diff
@@ -644,7 +644,7 @@ async function tryPrAutomerge(
       }
 
       const desiredStatus = 'can_be_merged';
-      const retryTimes = 5;
+      const retryTimes = 35; // results in max. 5 min. timeout if no pipeline created
```

Comment:
instead of hammering the API I would increase the delay exponentially

Reply:
But that is what it is doing here: https://github.com/fullstackcreatives/renovate/blob/b579f8febe8bfce3d27a1f9fb4125dea38aa23ac/lib/modules/platform/gitlab/index.ts#L661

Reply:
so 35 tries is far more longer than 5min, because all previous attempts accumulate

Reply:
it will delay ~87 minutes on worst case

Reply:
@rarkins we should revert or reduce this

Reply:
10 tries will be ~7.64 minutes delay

Reply:
8 tries are 5 minutes

Reply:
- https://github.com/renovatebot/renovate/pull/25994

Reply:
No, 8 tries is 18000 ms = 18 sec. It calculates as follows: 500 + 1000 + 1500 + 2000 + 2500 + 3000 + 3500 + 4000 = 18000

This is how it was calculated already before my change. It makes sense because normally starting up pipeline is fast so waiting time here would be minimized.
In our large gitlab instance we need some more time and so we agreed upon the 5 minutes which means 35 retries with the above logic.